Sources/PointFree/Account/Account.swift

```diff
@@ -13,7 +13,17 @@ let accountResponse =
   filterMap(require1 >>> pure, or: loginAndRedirect)
     <| fetchAccountData
     >-> writeStatus(.ok)
-    >-> respond(accountView.contramap(lower), layout: simplePageLayout(title: "Account", currentUser: get5))
+    >-> map(lower)
```

Comment:
now instead of `myView.contramap(lower)` you'll probably insert this transformation just before the layout `respond` middleware.